Powerops 2694 use fdm in shop trigger (#427)
cognitedata/power-ops-sdk: CHANGELOG.md

- `Fixed` for any bug fixes.
- `Security` in case of vulnerabilities.

## [0.99.0] - 2024-10-07
### Added

> * Support for triggering Shop using the ShopCase Data Model

Reviewer: ```suggestion
* Support for triggering Shop using a ShopCase instance
```
